### ISSUE TITLE
изображение не обновляется

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#461]
```
в последней версии программы при 
перемещении по странице изображение не 
обновляется. Если подвигать шкалу 
масштаба, то изображение обновляется.
Android 2.3.7, Samsung Galaxy ace.
```

Original issue reported on code.google.com by `podkopae...@gmail.com` on 20 Feb 2013 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
Не поясните, что значит "не обновляется"? 
Может скриншоты добавите?
```

Original comment by `Andrei.K...@gmail.com` on 20 Feb 2013 at 7:44

**Comment by GoogleCodeExporter** [comment]
```
если страницу просматривать не полностью 
(масштаб >1), то при перемещении по странице 
остается белое пятно (та область, которая 
должна показываться). если масштаб 
подвигать, то изображение начинает 
прорисовываться. Проблема в том, что 
несколько раз все нормально, а потом может 
не прорисоваться. 
Скриншот постараюсь сделать.
```

Original comment by `podkopae...@gmail.com` on 20 Feb 2013 at 4:51

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.0
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 10:49
- Changed state: **Fixed**
